src/components/Form/ForeignBornDocuments/ForeignBornDocuments.jsx

```diff
@@ -290,6 +290,9 @@ export default class ForeignBornDocuments extends ValidationElement {
             label="Document Number"
             {...this.props.DocumentNumber}
             className="foreign-born-document-number"
+            maxlength="30"
+            pattern="^[a-zA-Z0-9]*$"
+            prefix={"alphanumeric"}
```

Comment:
I think we can omit the `{}` in each of these instances since we aren't trying to interpolate a variable, the quoted string should be enough